eng/performance/BulkLoad-Performance.ps1

```diff
@@ -0,0 +1,31 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+<#
+.DESCRIPTION
+    Measure Bulk Load Performance
+#>
+param(
+    [ValidateSet('GrandBend', 'PartialGrandBend')]
```

Comment:
Should there be a script for Southridge?

Reply:
I considered this to be part of [RND-573](https://tracker.ed-fi.org/browse/RND-573), but I can add the option here even that the script is not yet present